.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Feature enhancement request
+about: Suggest an idea for this project
+
+---
+
+/kind feature
```

Comment:
This should be `@miq-bot add-label enhancement`

Reply:
Done